rom/laghos.cpp

```diff
@@ -390,6 +406,98 @@ int main(int argc, char *argv[])
 
     NormType normtype = localmap[normtype_char];
 
+    std::vector<double> paramPoints;
+
+    CAROM::GreedyParameterPointSelector* parameterPointGreedySelector = NULL;
+
+    // If using the greedy algorithm, initialize the parameter point greedy selector.
+    if (rom_build_database)
+    {
```

Comment:
This could be a function, to simplify `main()`. Also the `rom_use_database` case.